pkg/kubelet/kubelet_node_status_test.go

```diff
@@ -1648,21 +1648,32 @@ func TestSetVolumeLimits(t *testing.T) {
 			expectedVolumeKey: util.AzureVolumeLimitKey,
 			expectedLimit:     16,
 		},
+		{
+			name:              "when no cloudprovider is present",
+			cloudProviderName: "",
+			expectedVolumeKey: util.AzureVolumeLimitKey,
```

Comment:
How is this specific key picked up when cloud provider is set to empty?

Reply:
We could have tested any cloudprovider. I just chose to test Azure.

Reply:
also there's a loop in setVolumeLimits that goes through all the plugins so we are just picking one to test here.